src/backend/tests/it/utils/assertion.rs

```diff
@@ -1,5 +1,6 @@
 use shared::types::custom_token::CustomToken;
 use shared::types::token::UserToken;
+use shared::types::TokenTimestamp;
 
```

Comment:
Suggestion:

If you add these traits to UserToken:
```
    #[derive(CandidType, Deserialize, Clone, Debug, Eq, PartialEq)]
    pub struct UserToken {
```
you can do `assert_eq!(&one_token_list, &another_token_list, "My lists aren't the same!")` without needing a dedicated comparison function.

If the default definition of `Eq` does not suit, it is possible to define a custom one.

Reply:
Sure we can do that in another PR. Original did not wanted to change the existing struct but, we are passed that now ;)